common/config/playwright/playwright.config.common.ts

```diff
@@ -73,10 +73,7 @@ export const config: PlaywrightTestConfig<TestOptions> = {
     },
     // components tests use toHaveScreenshot
     toHaveScreenshot: {
-      maxDiffPixels: 4,
-      // make stricter comparison for colors, default value was 0.2
-      // which didn't catch some color differences for gray colors
-      threshold: 0
+      maxDiffPixels: 1
```

Comment:
Double check with calling side that this change won't negatively effect any of their tests

Reply:
WIll do.
Also, tested this locally and with CI. Also, the `maxDiffPixels` was updated by me a week or so ago because of the RTE tests.

Reply:
Posted in the web ui chat